AmplifyTools/AmplifyXcode/README.md

```diff
@@ -0,0 +1,3 @@
+# AmplifyXcode
+
+A description of this package.
```

Comment:
Make sure we expand this (separate PR is preferable) before releasing.

Specifically, let's note that even though much of the source has Public accessibility, this tool is under active development and the API could change without warning. Basically, I want us to reserve the right to change the implementation details of this tool without worrying about breaking changes. At some future time, we'll freeze the API and be more rigorous about protecting against breaking changes.

Reply:
Of course, thanks! A follow up PR is on the way